forem/forem
# fix: prevent dashboard following lists rendering stale content

## What type of PR is this? (check all applicable)


## Description

Prevents the 'Following users' list being appended to the 'Following organisations' list.

## Related Tickets & Documents



- Closes #18509
- Closes #19483

## QA Instructions, Screenshots, Recordings

Tested on a Mac with the latest Chrome version, but I can't see this being an issue with older browsers

For QA, follow the repro steps in #18509 and notice how the issue no longer occurs :)

### UI accessibility concerns?

N/A

## Added/updated tests?

- [x] No, and this is why: I couldn't see where the existing tests for this behaviour are (if there are any). Happy to add tests if given some guidance :)
